libkbfs/keybase_service_base.go

```diff
@@ -414,7 +441,10 @@ func (k *KeybaseServiceBase) LoadUserPlusKeys(ctx context.Context,
 // KeybaseServiceBase.
 func (k *KeybaseServiceBase) LoadTeamPlusKeys(
 	ctx context.Context, tid keybase1.TeamID) (TeamInfo, error) {
-	// No caching until the invalidations are ready.
+	cachedTeamInfo := k.getCachedTeamInfo(tid)
+	if cachedTeamInfo.Name != libkb.NormalizedUsername("") {
```

Comment:
I guess we can't cache bad team IDs since there's no way to tell the API server to notify us. But have we thought about doing that? Would it even be worth it?